### PR TITLE
Benchmark pallet-cc-authorities-noting

### PR DESCRIPTION
**Pull request by tmpolaczyk**

### CODE REVIEW

**Comment by tmpolaczyk** [comment]
I don't understand why is CI failing, I run the same command locally and it works:

```
cargo check --features=runtime-benchmarks --release --all
```

**Comment by tmpolaczyk** [comment]
Thanks @girazoki, not sure what happened here. It was working for me locally because I had modified the files in the `.cargo` dir, but not sure why the Cargo.lock had changes.

File: container-chains/pallets/authorities-noting/src/benchmarks.rs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            orchestrator_chain_state: orchestrator_proof,
+        };
+
+        T::RelayChainStateProvider::set_current_relay_chain_state(cumulus_pallet_parachain_system::RelayChainState {
+            state_root: relay_root,
+            number: 0,
+        });
+        #[cfg(test)]
+        {
+            // set_current_relay_chain_state doesn't work in tests, we need to write the MOCK_RELAY_ROOT_KEY.
```
Comment (girazoki) [inline container-chains/pallets/authorities-noting/src/benchmarks.rs:87]: Why wouldnt it? can we make it such that the `RelayChainStateProvider` set_current_relay_chain_state matches what you are doing here?

Comment (tmpolaczyk) [reply container-chains/pallets/authorities-noting/src/benchmarks.rs:87]: We would need to modify the `MockRelayChainStateProvider`.

**Pull request merged**